UNITTESTS/target_h/platform/mbed_retarget.h

```diff
@@ -374,9 +374,15 @@ struct stat {
 
     off_t     st_size;    ///< Size of file in bytes
 
+#ifndef st_atime
```

Comment:
what is this? is this some macro hackery?

Reply:
Yes a macro `st_atime` is defined in MacOS. I can add a comment to make it more explicit.